tests/network/user_defined_network/test_user_defined_network_passt.py

```
@@ -82,24 +82,28 @@ def passt_running_vm_pair(
 @pytest.mark.ipv4
 @pytest.mark.single_nic
 @pytest.mark.polarion("CNV-12427")
-def test_passt_connectivity_is_preserved_during_client_live_migration(passt_enabled_in_hco, passt_running_vm_pair):
+def test_passt_connectivity_is_preserved_during_client_live_migration(
+    admin_client, passt_enabled_in_hco, passt_running_vm_pair
```

Ruff (0.15.14)

[warning] Unused function argument: `passt_enabled_in_hco`

(ARG001)